### PR TITLE
New version: StefanBerg.NINA Version 3.0.0.9001

### PR DESCRIPTION
**Pull request by isbeorn**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/145109)

### CODE REVIEW

File: manifests/s/StefanBerg/NINA/3.0.0.9001/StefanBerg.NINA.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+PackageIdentifier: StefanBerg.NINA
+PackageVersion: 3.0.0.9001
+MinimumOSVersion: 10.0.0.0
+Installers:
+  - Architecture: x64
+    InstallerType: burn
+    InstallerUrl: https://f002.backblazeb2.com/file/ninasetup/Releases/3.0.0.9001/NINASetupBundle_3.0.0.9001.exe
+    InstallerSha256: AD92A2FEA648A0CE6D9B28CDDFFFB6DCD60268FCD2FE30549C48EEFC343DD052
+ManifestType: installer
+ManifestVersion: 1.2.0
```
Comment (stephengillie) [inline manifests/s/StefanBerg/NINA/3.0.0.9001/StefanBerg.NINA.installer.yaml:10]:
```suggestion
ManifestVersion: 1.6.0
```

File: manifests/s/StefanBerg/NINA/3.0.0.9001/StefanBerg.NINA.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+PackageIdentifier: StefanBerg.NINA
+PackageVersion: 3.0.0.9001
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.2.0
```
Comment (stephengillie) [inline manifests/s/StefanBerg/NINA/3.0.0.9001/StefanBerg.NINA.yaml:5]:
```suggestion
ManifestVersion: 1.6.0
```

File: manifests/s/StefanBerg/NINA/3.0.0.9001/StefanBerg.NINA.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+ShortDescription: N.I.N.A. - Nighttime Imaging 'N' Astronomy is a free open source project dedicated to deep sky astrophotography.
+Description: N.I.N.A. is a software designed for all deep sky object imagers. Whether you're new to the world of Deep Sky Object imaging or an experienced veteran, this projects goal is to make your image capture easier, faster, and more convenient.
+ReleaseNotesUrl: https://f002.backblazeb2.com/file/ninasetup/Releases/3.0.0.9001/RELEASE_NOTES.html
+Documentations: 
+  - DocumentLabel: N.I.N.A. Documentation
+    DocumentUrl: https://nighttime-imaging.eu/docs/master/site/
+  - DocumentLabel: Issue Tracker
+    DocumentUrl: https://bitbucket.org/Isbeorn/nina/issues?status=new&status=open
+ManifestType: defaultLocale
+ManifestVersion: 1.2.0
```
Comment (stephengillie) [inline manifests/s/StefanBerg/NINA/3.0.0.9001/StefanBerg.NINA.locale.en-US.yaml:21]:
```suggestion
ManifestVersion: 1.6.0
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @isbeorn,

We've depreciated this manifest version, but most fields are identical with the newest manifest version. In most cases, simply increasing the manifest version number is enough.

**Comment by isbeorn** [comment]
> Hi @isbeorn,
> 
> We've depreciated this manifest version, but most fields are identical with the newest manifest version. In most cases, simply increasing the manifest version number is enough.

Thank you for the feedback. I've adjusted the manifest version and re-ran the validation successfully with that

**Review by stephengillie — APPROVED** [review]

**Pull request merged**